head.html

```diff
@@ -69,15 +93,15 @@
     const insertInside = getSection(2);
     if (insertInside) {
       insertInside.classList.add('left');
-      const r = /^[Bb]y (.*)\n*(.*)$/gm.exec(insertInside.innerText);
+      const r = /^By (.*)\n*(.*)$/gmi.exec(insertInside.innerText);
       const author = r && r.length > 0 ? r[1] : null;
       const date = r && r.length > 1 ? r[2] : ''
       if (author) {
         // clear the content of the div and replace by avatar and text
         insertInside.innerHTML = '';
         const xhr = new XMLHttpRequest();
         const fileName = author.replace(/\s/gm, '-').toLowerCase();
-        const pageURL = '/authors/' + fileName + '.html';
+        const pageURL = getLink(pageTypes[1], author);
```

Comment:
`pageTypes[x]` is no so clear. Maybe we can replace the array by an object and do something like: 

`getLink(TYPES.AUTHOR, author);` 

or with single properties like:

`getLink(TYPE_AUTHOR, author);`

WDYT ?

Reply:
Note that this would be a more global change.

Reply:
Let's do that in a separate step, though.